toxic_timeout.go

```diff
@@ -8,6 +11,10 @@ type TimeoutToxic struct {
 	Enabled bool `json:"enabled"`
 	// Times in milliseconds
 	Timeout int64 `json:"timeout"`
+	// If true, use Toxicity
+	SometimesToxic bool `json:"sometimesToxic"`
+	// 'Toxicity' or probability of timing out 0..1
+	Toxicity float32 `json:"toxicity"`
```

Comment:
I think this was just because of compatibility. There isn't a way to specify a default value without changing the toxiproxy core.